feat: add battery health endpoint for cars

LukStankovic opened the pull request:
Feature request from https://github.com/tobiasehlert/teslamateapi/issues/347

tobiasehlert commented:
@LukStankovic, I think we should be able to use the `date_trunc` function ([psql docs](https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-TRUNC)) instead of that floor-extract-epoch instead? If that's the case that would be much better :)

tobiasehlert commented on src/v1_TeslaMateAPICarsBatteryHealth.go:
```diff
@@ -0,0 +1,324 @@
+		batteryHealthPercentage = (CurrentCapacity / MaxCapacity) * 100
+	} else {
+		batteryHealthPercentage = 100
+	}
```
Well, it sounds weird to say that the health is 100% if it can't be calculated. It like having a totally dead car being at it's best :D

Which dashboard and query was that shown? Maybe we can look into the gitblame history and see what made that implementation for that hehe

LukStankovic commented:
@tobiasehlert I changed it to `date_trunc` and also removed batteryHealth.BatteryHealthPercentage = 100, setting the default to 0 instead.

Would it maybe make more sense to use `NullFloat64` for `BatteryHealthPercentage`?

tobiasehlert commented:
My guess is that there should be no situation where the _MaxCapacity_ will be below 0, except in some special cases (guessing when it's a brand new car without milage or maybe some totalled-car situation). So I think we can just use float64 as for now, that should be totally fine @LukStankovic.
